### ISSUE TITLE
Confirm that the following information will cause the centos7.4 reboot

### ISSUE THREAD

**Issue by zqdlove** [moby/moby#35818]
**Description**

Confirm that the following information will cause the centos7.4 reboot

**Steps to reproduce the issue:**
1. It is difficult to reproduce
2. This happens only once

**Describe the results you received:**

```
system_msg_125.88.38.28-2017-15-160000664000175200017520047470370113215146270016335 0ustar  adminuseradminuserDec 15 23:02:05 125 dockerd-current: time="2017-12-16T12:02:05.343975951+08:00" level=error msg="Handler for POST /v1.24/containers/9f26e907c916/start returned error: Container already started"
Dec 15 23:02:05 125 dockerd-current: time="2017-12-16T12:02:05.395384576+08:00" level=info msg="{Action=start, Username=batchuser, LoginUID=1003, PID=17858}"
Dec 15 23:02:05 125 dockerd-current: time="2017-12-16T12:02:05.395525440+08:00" level=error msg="Handler for POST /v1.24/containers/9f26e907c916/start returned error: Container already started"
Dec 15 23:02:05 125 dockerd-current: time="2017-12-16T12:02:05.449416831+08:00" level=info msg="{Action=start, Username=batchuser, LoginUID=1003, PID=17906}"
Dec 15 23:02:05 125 dockerd-current: time="2017-12-16T12:02:05.449547708+08:00" level=error msg="Handler for POST /v1.24/containers/9f26e907c916/start returned error: Container already started"
Dec 15 23:02:05 125 dockerd-current: time="2017-12-16T12:02:05.501925887+08:00" level=info msg="{Action=start, Username=batchuser, LoginUID=1003, PID=17958}"
Dec 15 23:02:05 125 dockerd-current: time="2017-12-16T12:02:05.502045563+08:00" level=error msg="Handler for POST /v1.24/containers/9f26e907c916/start returned error: Container already started"
Dec 15 23:02:05 125 dockerd-current: time="2017-12-16T12:02:05.552413533+08:00" level=info msg="{Action=start, Username=batchuser, LoginUID=1003, PID=18004}"
Dec 15 23:02:05 125 dockerd-current: time="2017-12-16T12:02:05.552520882+08:00" level=error msg="Handler for POST /v1.24/containers/9f26e907c916/start returned error: Container already started"
Dec 15 23:02:05 125 dockerd-current: time="2017-12-16T12:02:05.606410837+08:00" level=info msg="{Action=start, Username=batchuser, LoginUID=1003, PID=18053}"
Dec 15 23:02:05 125 dockerd-current: time="2017-12-16T12:02:05.606582109+08:00" level=error msg="Handler for POST /v1.24/containers/9f26e907c916/start returned error: Container already started"
Dec 15 23:02:05 125 dockerd-current: time="2017-12-16T12:02:05.657079626+08:00" level=info msg="{Action=start, Username=batchuser, LoginUID=1003, PID=18097}"
Dec 15 23:02:05 125 dockerd-current: time="2017-12-16T12:02:05.657227636+08:00" level=error msg="Handler for POST /v1.24/containers/9f26e907c916/start returned error: Container already started"
Dec 16 12:04:52 125 systemd: Started Run docker-cleanup every hour.
Dec 16 12:04:52 125 systemd: Starting Run docker-cleanup every hour.
Dec 16 12:04:52 125 container-storage-setup: ERROR: Docker has been previously configured for use with devicemapper graph driver. Not creating a new thin pool as existing docker metadata will fail to work with it. Manual cleanup is required before this will succeed.
Dec 16 12:04:52 125 container-storage-setup: INFO: Docker state can be reset by stopping docker and by removing /var/lib/docker directory. This will destroy existing docker images and containers and all the docker metadata.
Dec 16 12:04:52 125 systemd: docker-storage-setup.service: main process exited, code=exited, status=1/FAILURE
Dec 16 12:04:52 125 systemd: Unit docker-storage-setup.service entered failed state.
Dec 16 12:04:52 125 systemd: docker-storage-setup.service failed.
Dec 16 12:05:02 125 dockerd-current: time="2017-12-16T12:05:02.570106454+08:00" level=info msg="libcontainerd: new containerd process, pid: 2369"
Dec 16 12:05:03 125 dockerd-current: time="2017-12-16T12:05:03.767466736+08:00" level=warning msg="devmapper: Usage of loopback devices is strongly discouraged for production use. Please use `--storage-opt dm.thinpooldev` or use `man docker` to refer to dm.thinpooldev section."
Dec 16 12:05:03 125 dockerd-current: time="2017-12-16T12:05:03.794604805+08:00" level=warning msg="devmapper: Base device already exists and has filesystem xfs on it. User specified filesystem  will be ignored."
Dec 16 12:05:03 125 dockerd-current: time="2017-12-16T12:05:03.813990604+08:00" level=info msg="[graphdriver] using prior storage driver \"devicemapper\""
Dec 16 12:05:03 125 dockerd-current: time="2017-12-16T12:05:03.888834939+08:00" level=warning msg="Docker could not enable SELinux on the host system"
Dec 16 12:05:03 125 dockerd-current: time="2017-12-16T12:05:03.914941801+08:00" level=info msg="Graph migration to content-addressability took 0.00 seconds"
Dec 16 12:05:03 125 dockerd-current: time="2017-12-16T12:05:03.915493340+08:00" level=info msg="Loading containers: start."
Dec 16 12:05:03 125 dockerd-current: .time="2017-12-16T12:05:03.917576304+08:00" level=warning msg="libcontainerd: client is out of sync, restore was called on a fully synced container (9f26e907c9162f2d3fcda99632d9427d4010716734b4027da2ff32e43c5ecf68)."
Dec 16 12:05:03 125 dockerd-current: time="2017-12-16T12:05:03.917987390+08:00" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/9f26e907c9162f2d3fcda99632d9427d4010716734b4027da2ff32e43c5ecf68/shm: invalid argument"
Dec 16 12:05:03 125 dockerd-current: time="2017-12-16T12:05:03.922129678+08:00" level=error msg="devmapper: Error unmounting device f1a199b8f018b15bb93c288f3ea431aedd56650ad821813c018d8481e2f53682: invalid argument"
Dec 16 12:05:03 125 dockerd-current: time="2017-12-16T12:05:03.922154530+08:00" level=error msg="Error unmounting container 9f26e907c9162f2d3fcda99632d9427d4010716734b4027da2ff32e43c5ecf68: invalid argument"
Dec 16 12:05:03 125 dockerd-current: time="2017-12-16T12:05:03.965649815+08:00" level=info msg="Firewalld running: false"
Dec 16 12:05:04 125 NetworkManager[1117]: <info>  [1513397104.0358] manager: (docker0): new Bridge device (/org/freedesktop/NetworkManager/Devices/7)
Dec 16 12:05:04 125 NetworkManager[1117]: <info>  [1513397104.0833] device (docker0): state change: unmanaged -> unavailable (reason 'connection-assumed') [10 20 41]
Dec 16 12:05:04 125 kernel: IPv6: ADDRCONF(NETDEV_UP): docker0: link is not ready
Dec 16 12:05:04 125 NetworkManager[1117]: <info>  [1513397104.0850] ifcfg-rh: add connection in-memory (0d97e184-f253-4eae-8232-5a78794bd0ae,"docker0")
Dec 16 12:05:04 125 NetworkManager[1117]: <info>  [1513397104.0856] device (docker0): state change: unavailable -> disconnected (reason 'connection-assumed') [20 30 41]
Dec 16 12:05:04 125 NetworkManager[1117]: <info>  [1513397104.0865] device (docker0): Activation: starting connection 'docker0' (0d97e184-f253-4eae-8232-5a78794bd0ae)
Dec 16 12:05:04 125 NetworkManager[1117]: <info>  [1513397104.0871] device (docker0): state change: disconnected -> prepare (reason 'none') [30 40 0]
Dec 16 12:05:04 125 NetworkManager[1117]: <info>  [1513397104.0875] device (docker0): state change: prepare -> config (reason 'none') [40 50 0]
Dec 16 12:05:04 125 NetworkManager[1117]: <info>  [1513397104.0878] device (docker0): state change: config -> ip-config (reason 'none') [50 70 0]
Dec 16 12:05:04 125 NetworkManager[1117]: <info>  [1513397104.0899] device (docker0): state change: ip-config -> ip-check (reason 'none') [70 80 0]
Dec 16 12:05:04 125 NetworkManager[1117]: <info>  [1513397104.0904] device (docker0): state change: ip-check -> secondaries (reason 'none') [80 90 0]
Dec 16 12:05:04 125 NetworkManager[1117]: <info>  [1513397104.0906] device (docker0): state change: secondaries -> activated (reason 'none') [90 100 0]
Dec 16 12:05:04 125 NetworkManager[1117]: <info>  [1513397104.0914] device (docker0): Activation: successful, device activated.
Dec 16 12:05:04 125 nm-dispatcher: req:8 'up' [docker0]: new request (3 scripts)
Dec 16 12:05:04 125 nm-dispatcher: req:8 'up' [docker0]: start running ordered scripts...
Dec 16 12:05:04 125 dockerd-current: time="2017-12-16T12:05:04.132491588+08:00" level=info msg="Removing stale sandbox 68ec290cb1de49baf2c357f3a164484ce619ccfc8226c34f4600091fc4d5c16a (9f26e907c9162f2d3fcda99632d9427d4010716734b4027da2ff32e43c5ecf68)"
Dec 16 12:05:04 125 dockerd-current: time="2017-12-16T12:05:04.172077572+08:00" level=info msg="Default bridge (docker0) is assigned with an IP address 172.17.0.0/16. Daemon option --bip can be used to set a preferred IP address"
Dec 16 12:05:04 125 dockerd-current: time="2017-12-16T12:05:04.215755175+08:00" level=info msg="Loading containers: done."
Dec 16 12:05:04 125 dockerd-current: time="2017-12-16T12:05:04.215849883+08:00" level=info msg="Daemon has completed initialization"
Dec 16 12:05:04 125 dockerd-current: time="2017-12-16T12:05:04.215867745+08:00" level=info msg="Docker daemon" commit="0fdc778/1.12.6" graphdriver=devicemapper version=1.12.6
Dec 16 12:05:04 125 dockerd-current: time="2017-12-16T12:05:04.222076473+08:00" level=info msg="API listen on /var/run/docker.sock"
Dec 16 12:05:04 125 dockerd-current: time="2017-12-16T12:05:04.265899134+08:00" level=info msg="{Action=start, LoginUID=4294967295, PID=4073}"
Dec 16 12:05:04 125 dockerd-current: time="2017-12-16T12:05:04.353844153+08:00" level=warning msg="error locating sandbox id 68ec290cb1de49baf2c357f3a164484ce619ccfc8226c34f4600091fc4d5c16a: sandbox 68ec290cb1de49baf2c357f3a164484ce619ccfc8226c34f4600091fc4d5c16a not found"
Dec 16 12:05:04 125 dockerd-current: time="2017-12-16T12:05:04.353890249+08:00" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/9f26e907c9162f2d3fcda99632d9427d4010716734b4027da2ff32e43c5ecf68/shm: invalid argument"
Dec 16 12:05:04 125 dockerd-current: time="2017-12-16T12:05:04.353918698+08:00" level=error msg="devmapper: Error unmounting device f1a199b8f018b15bb93c288f3ea431aedd56650ad821813c018d8481e2f53682: invalid argument"
Dec 16 12:05:04 125 dockerd-current: time="2017-12-16T12:05:04.353930303+08:00" level=error msg="Error unmounting container 9f26e907c9162f2d3fcda99632d9427d4010716734b4027da2ff32e43c5ecf68: invalid argument"
Dec 16 12:05:04 125 dockerd-current: time="2017-12-16T12:05:04.354254277+08:00" level=error msg="Handler for POST /v1.24/containers/9f26e907c916/start returned error: devmapper: Error mounting '/dev/mapper/docker-253:0-716870-f1a199b8f018b15bb93c288f3ea431aedd56650ad821813c018d8481e2f53682' on '/var/lib/docker/devicemapper/mnt/f1a199b8f018b15bb93c288f3ea431aedd56650ad821813c018d8481e2f53682': invalid argument"
Dec 16 12:05:04 125 rc.local: Error response from daemon: devmapper: Error mounting '/dev/mapper/docker-253:0-716870-f1a199b8f018b15bb93c288f3ea431aedd56650ad821813c018d8481e2f53682' on '/var/lib/docker/devicemapper/mnt/f1a199b8f018b15bb93c288f3ea431aedd56650ad821813c018d8481e2f53682': invalid argument
Dec 16 12:05:04 125 dockerd-current: time="2017-12-16T12:05:04.413930088+08:00" level=info msg="{Action=start, LoginUID=4294967295, PID=4249}"
```
**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**

issue happens only occasionally

**Output of `docker version`:**

```
$sudo docker -v
Docker version 1.12.6, build ec8512b/1.12.6
```

**Output of `docker info`:**

```
$sudo docker info
Containers: 1
 Running: 0
 Paused: 0
 Stopped: 1
Images: 4
Server Version: 1.12.6
Storage Driver: devicemapper
 Pool Name: docker-8:5-255288-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 10.74 GB
 Backing Filesystem: xfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 686.7 MB
 Data Space Total: 107.4 GB
 Data Space Available: 106.7 GB
 Metadata Space Used: 1.229 MB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.146 GB
 Thin Pool Minimum Free Space: 10.74 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: true
 Deferred Deletion Enabled: true
 Deferred Deleted Device Count: 0
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 WARNING: Usage of loopback devices is strongly discouraged for production use. Use `--storage-opt dm.thinpooldev` to specify a custom block storage device.
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.135-RHEL7 (2016-09-28)
Logging Driver: journald
Cgroup Driver: systemd
Plugins:
 Volume: local
 Network: overlay host bridge null
Swarm: inactive
Runtimes: docker-runc runc
Default Runtime: docker-runc
Security Options: seccomp
Kernel Version: 3.10.0-229.el7.x86_64
Operating System: CentOS Linux 7 (Core)
OSType: linux
Architecture: x86_64
Number of Docker Hooks: 3
CPUs: 32
Total Memory: 62.35 GiB
Name: 1.1.1.1
ID: B3I4:6VDG:LR62:VKEQ:26Y5:ZGWS:ZQXD:V7TW:57E4:KKW2:PQCL:4DOT
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: bridge-nf-call-iptables is disabled
WARNING: bridge-nf-call-ip6tables is disabled
Insecure Registries:
 127.0.0.0/8
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

physical

**Comment by thaJeztah** [comment]
Docker 1.12.6 has reached end-of-life in March, and is no longer maintained. From the output of your log-files; the `dockerd-current` binary is likely a binary that's installed from the Red Hat fork of Docker, which is not maintained here (issues with that package should be reported with Red Hat).

I'd recommend installing a current version of the official packages (see https://docs.docker.com/engine/installation/linux/docker-ce/centos/ for instructions)

I'm closing this issue because of the above, but feel free to continue the conversation

**Comment by zqdlove** [comment]
Thank you
